graphql/schema.graphql

```diff
@@ -71,6 +71,7 @@ type Query {
   taskQueueDistros: [TaskQueueDistro!]!
   buildBaron(taskId: String!, execution: Int!): BuildBaron!
   bbGetCreatedTickets(taskId: String!): [JiraTicket!]!
+  annotation(taskId: String!, execution: Int!): Annotation
```

Comment:
I think the schema design can benefit from moving this query to a conditionally resolved field underneath the Task gql type. A developer who is new to our schema can understand how to query task annotations by looking at the Task gql type instead of reading through our growing list of queries. This will also allow us to omit the `taskId` and `execution` since the base model for the Task gql type includes the execution and the taskId